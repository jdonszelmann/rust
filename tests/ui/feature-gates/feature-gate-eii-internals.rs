#![crate_type = "rlib"]
#![feature(decl_macro)]
#![feature(rustc_attrs)]

#[eii_macro_for(bar)] //~ ERROR use of unstable library feature `eii_internals`
#[rustc_builtin_macro(eii_macro)]
macro foo() {

}

unsafe extern "Rust" {
    safe fn bar(x: u64) -> u64;
}
