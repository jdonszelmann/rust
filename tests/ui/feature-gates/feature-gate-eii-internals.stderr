error[E0658]: use of unstable library feature `eii_internals`
  --> $DIR/feature-gate-eii-internals.rs:5:3
   |
LL | #[eii_macro_for(bar)]
   |   ^^^^^^^^^^^^^
   |
   = help: add `#![feature(eii_internals)]` to the crate attributes to enable
   = note: this compiler was built on YYYY-MM-DD; consider upgrading it if it is out of date

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0658`.
